railties/lib/rails/commands/console.rb

```diff
@@ -7,6 +7,12 @@ module Rails
   class Console
     include ConsoleHelper
 
+    module BacktraceCleaner
+      def filter_backtrace(bt)
+        (v = super) && Rails.backtrace_cleaner.filter([v]).first
```

Comment:
A small cosmetic nit: we can avoid reverencing the named argument, as we are not using it (may avoid Ruby warnings). We can also make the condition a bit more explicit.

```ruby
def filter_backtrace(_)
  if v = super
    Rails.backtrace_cleaner.filter([v]).first
  end
end
```